components/FolderButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { styled } from "styled-components";
+
+type props = {
+  children: React.ReactNode;
+  request: () => void;
+  isActive: boolean;
+};
+
+const FolderButton = ({ children, request, isActive }: props) => {
+  return (
+    <ButtonStyle onClick={() => request()} isActive={isActive}>
```

Comment:
스타일드 컴포넌트에 넘기는 props의 경우에는 달러사인으로 넘겨주면 좋은데요!
요것도 찾아보시면 좋을 것 같아요~
```suggestion
    <ButtonStyle onClick={() => request()} $isActive={isActive}>
```